### DISCUSSION TITLE
Add Heal Group cooldown to Timers plugin

### DISCUSSION THREAD

**Discussion by Nope1252** [runelite/runelite#16511] [Ideas]
It'd be a nice addition for alting and bossing

**Comment by Nightfirecat** [comment]
Is there a unique graphic / chat message / var change for heal other? Does your spellbook have a dimmed icon when it is cast?
We can certainly add this, but the particular method by which it's added depends on some of those factors.

**Reply by Nope1252** [reply]
The spell icon is dimmed the same way the veng icons are when they're on cooldown. Heal group is on a 90 second cooldown though, unlike veng. 
![VfkME9y](https://user-images.githubusercontent.com/96698029/233509596-680d8a2a-8cab-4b9d-b74a-cab60a8d770a.png)

**Comment by Nightfirecat** [comment]
Added in 028e1c59e25a2dfc9dd77f07fd63d06cefdcf733
